functions.php

```diff
@@ -64,3 +63,40 @@ function addItemToHTML(array $dataFromQuery): string
     }
     return $result;
 }
+
+/**
+ * extracts the brands and their ids from the database and returns them in an associative array
+ * @param PDO $db, the database to extract from
+ *
+ * @return array array of the brands and their ids
+ */
+function extract_brands_from_db(PDO $db): array {
+    $query = $db->prepare('SELECT `id`,`nameOfBrand` FROM `brands`;');
+    $query->execute();
+    return $query->fetchAll();
+}
+
+/**
+ * extracts the regions and their ids from
+ * @param PDO $db, the database to extract from the database and returns them in an associative array
+ *
+ * @return array array of the regions and their ids
+ */
+function extract_regions_from_db(PDO $db): array {
+    $query = $db->prepare('SELECT `id`,`region` FROM `regions`;');
+    $query->execute();
+    return $query->fetchAll();
+}
+
+/**
+ * adds the array of values to the db after changing the data types to match the db structure
+ * @param array $postArray, the array of values you want to add into the db
+ * @param PDO $db , the db you want to add the values to
+ */
+function add_new_item_to_db(array $postArray,PDO $db) {
+    settype($postArray['brand'], "integer");
```

Comment:
why?

Reply:
i thought i would need to change the data type to insert into the db but didn't think to check if it would work with strings, changed it now